web/client/plugins/Save.jsx

```diff
@@ -26,7 +27,8 @@ export default createPlugin('Save', {
             mapInfoSelector,
             (show, resource) => ({show, resource})),
         {
-            onClose: toggleControl.bind(null, 'mapSave', false)
+            onClose: toggleControl.bind(null, 'mapSave', false),
+            onMapSaveError: mapSaved
```

Comment:
This looks a little strange to me? mapSaved bind to mapSaveError?

Reply:
yes, it is used to [reset errors](https://github.com/geosolutions-it/MapStore2/blob/master/web/client/reducers/config.js#L118) when edit properties modal is closed..

going to change the name to
```suggestion
            onResetMapSaveError: mapSaved
```

Reply:
Create a specific action for it, otherwise is confusing

Reply:
Or you create a RESET_ERRORS action or you explain that you are using the mapSaved action to reset error. I'm not sure if you pass the map or not. Please explain and find out a better name.

onResetMapSaveError doesn't make to much sense. what it means? "on Reset, MapSaveError"/ on reset map, save error"...
handlers name with "on" means that they are called in sequence of an event. 
onReset means that is called when button "Reset" is clicked, or a "reset" action is performed by the user, anyway,
onPan means the handler is called when the map pans.

Instead, if the property is instead a function that have to be called to have some effect, you can call it "reset". 
"resetError" will reset the error.

Reply:
i could create another constant in config.js action file that does the same thing as MAP_SAVED
i wanted to reuse it, gonna create it anyway and change the name of the action passed to the plugin to resetErrors

Reply:
*correction gonna call it **clearErrors**